src/components/LayoutFooter/Footer.js

```diff
@@ -176,4 +135,19 @@ const Footer = () => (
   </footer>
 );
 
-export default Footer;
+export default () => (
+  <StaticQuery
+    query={graphql`
+      query {
+        site {
+          siteMetadata {
+            contact {
+              email
+            }
```

Comment:
Maybe the footer links should go into siteMetadata too?

Reply:
done

Reply:
Thanks @eouin